CONTRIBUTING.md

```diff
@@ -21,35 +21,39 @@ pull requests.
   client for the language server
 * `test-workspace` directory for manually testing the plugin locally
 
-## Running locally
-
-First you need to have the following applications installed
+## Running a local version of the server
+You will need the following applications installed:
 
 * `git`
 * `sbt`
 * Visual Studio Code (`code` from the console)
-* `npm`
-
-To try out metals locally, it's best to keep two open terminal windows.
 
 ```
-git clone https://github.com/scalameta/metals.git
-cd metals
-
-########
-# Step 1
-########
 sbt
-> metals/publishLocal # publish your latest changes locally
+> metals/publishLocal     # publish your latest changes locally
                           # keep this sbt shell session open, and
                           # re-run publishLocal every time you
                           # edit *.scala sources.
 > ~testWorkspace/test:compile # compile the sources in test-workspace
```

Comment:
How about making these comments a normal text? Something like this:

----

Publish your latest changes locally. Keep this sbt shell session open and re-run `publishLocal` every time you edit `*.scala` sources:

```
> metals/publishLocal
```

----

Also, shouldn't it be just `publishLocal` to publish also dependent modules?

Reply:
Yeah, I think text is better too (I copy pasted it from the previous guide)

Doesn't `metals/publishLocal` effectively publishes also its dependencies? If not, it should definitely be `publishLocal`